progressbar.go

```diff
@@ -449,31 +472,40 @@ func renderProgressBar(c config, s state) (int, error) {
 	}
 
 	// add on bytes string if max bytes option was set
-	kbPerSecond := averageRate / float64(c.max) * float64(c.maxBytes) / 1000.0
+	// rolling average kb/sec
+	kbPerSecond := averageRate / 1024.0
+	// float64(p.state.currentBytes) / 1000.0 / s.SecondsSince
 
 	if kbPerSecond > 1000.0 {
```

Comment:
Since this is always activated now, it should check if OptionShowBytes is toggled.